Update ApplicationProcess.js

germonprez opened the pull request:
Updating the text associated with the Application Process text. 

Signed-off-by: Matt Germonprez <germonprez@gmail.com>

@kaxada @ElizabethN

kaxada commented on Website-footer/Footer.html:
```diff
@@ -0,0 +1,74 @@
+</head>
+<body>
+
+  <footer class="footer">
```
@Delight362 thank you for this contribution. everything works just fine but the issue we are using react so you have to place all these changes in a react component under the src/components folder with its HTML and we are good to go. Let me know if you need help in doing that. Otherwise, this is impressive. Sorry for taking all this long to review this but let's get this started and finished

kaxada commented on Website-footer/Footer1.css:
```diff
@@ -0,0 +1,158 @@
+	font-weight: 500;
+	position: relative;
+}
+/**.footer-col h4::before{
```
@Delight362 make sure you don't leave commented code in the final submission

kaxada reviewed (changes requested):
@Delight362 your contribution is awesome and apologies for taking long to make reviews. I request you make the footer tag a react component and all the related CSS and then resubmit this. I am delighted though

kaxada commented:
@Delight362 I am closing this PR because the changes I suggested were addressed in this PR https://github.com/badging/badging-website/pull/48. However, there is some polishing that needs to be made like the font style and Mobile responsiveness. As soon as the PR is merged. I request you build onto it with some your changes to make the footer look better

Delight362 commented:
Okay then

On Wed, Dec 7, 2022, 9:07 AM Kaxada ***@***.***> wrote:

> @Delight362 <https://github.com/Delight362> I am closing this PR because
> the changes I suggested were addressed in this PR #48
> <https://github.com/badging/badging-website/pull/48>. However, there is
> some polishing that needs to be made like the font style and Mobile
> responsiveness. As soon as the PR is merged. I request you build onto it
> with some your changes to make the footer look better
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/badging/badging-website/pull/29#issuecomment-1340554425>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXDHL6WMIETABLD4ILFST6DWMBAVDANCNFSM6AAAAAAQ6WZZRM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

germonprez committed:
Update ApplicationProcess.js

Updating the text associated with the Application Process text.

Signed-off-by: Matt Germonprez <germonprez@gmail.com>

@kaxada @ElizabethN
